org.envirocar.app/src/org/envirocar/app/views/preferences/BluetoothPairingPreference.java

```diff
@@ -64,6 +62,14 @@
 public class BluetoothPairingPreference extends DialogPreference {
     private static final Logger LOGGER = Logger.getLogger(BluetoothPairingPreference.class);
 
+
+    /**
+     *
+     */
+    public interface ShowSnackbarListener {
+        void showSnackbar(String text);
+    }
```

Comment:
Could you please, explain what is your intention about introducing an interface at this point, which is never implemented?

Reply:
The whole intention was to use consistent snack bars on all screens, but there were still some places where toasts were used. are there any disadvantages of using interfaces here please tell me It would be great learning.

Reply:
I guess there is no need for an interface, function `` showSnackbar `` is enough, will remove in new PR